pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -47,10 +47,10 @@ python3Packages.buildPythonApplication rec {
 
   crossFile = writeTextDir "cross-file.conf" ''
     [binaries]
```

Comment:
787b44f5a228ae7a765b6aa79fd67f42b73087f0 accidentally sneaked in?

Reply:
This is in my branch for testing; currently the tree is broken without it. See #48239.